test/boolean_complex_test.cpp

```diff
@@ -1030,4 +1031,102 @@ TEST(BooleanComplex, HullMask) {
   MeshGL mesh = ret.GetMeshGL();
 }
 
+TEST(BooleanComplex, SimpleOffset) {
+  std::string file = __FILE__;
+  std::string dir = file.substr(0, file.rfind('/'));
+  MeshGL seeds = ImportMesh(dir + "/models/" + "Generic_Twin_91.1.t0.glb");
+  EXPECT_TRUE(seeds.NumTri() > 10);
+  EXPECT_TRUE(seeds.NumVert() > 10);
+  // Unique edges
+  std::vector<std::pair<int, int>> edges;
+  for (size_t i = 0; i < seeds.NumTri(); i++) {
+    const int k[3] = {1, 2, 0};
+    for (const int j : {0, 1, 2}) {
+      int v1 = seeds.triVerts[i * 3 + j];
+      int v2 = seeds.triVerts[i * 3 + k[j]];
+      if (v2 > v1) edges.push_back(std::make_pair(v1, v2));
+    }
+  }
+  manifold::Manifold c;
+  // Vertex Spheres
+  Manifold sph = Manifold::Sphere(1, 8);
+  for (size_t i = 0; i < seeds.NumVert(); i++) {
+    vec3 vpos(seeds.vertProperties[3 * i + 0], seeds.vertProperties[3 * i + 1],
+              seeds.vertProperties[3 * i + 2]);
+    Manifold vsph = sph.Translate(vpos);
+    manifold::Manifold left = c;
+    manifold::Manifold right(vsph);
+    if (!right.NumTri()) continue;
+    c = left.Boolean(right, manifold::OpType::Add);
```

Comment:
is there a reason you don't use `c += right;`?

Reply:
That is partially left over from my initial wiring in BRL-CAD and partially my avoiding confusion in our code.  We use + for intersection.   No harm here though,  since the context is Manifold - I'll switch it once I get back to my computer.